bitsandbytes/diagnostics/cuda.py

```diff
@@ -38,6 +38,9 @@
     "nvcuda*.dll",  # Windows
 )
 
+if HIP_ENVIRONMENT:
+    CUDA_RUNTIME_LIB_PATTERNS = ("libamdhip64.so*",)
```

Comment:
For clarity, I'd make this

```
if HIP_ENVIRONMENT:
    CUDA_RUNTIME_LIB_PATTERNS = (...)
else:
    CUDA_RUNTIME_LIB_PATTERNS = (...)
```

so it's clear that this could get overridden.

Better yet, it could be moved to a function `def get_cuda_runtime_lib_patterns():` now that it's no longer constant.